packages/studio-be/src/studio/studio-router.ts

```diff
@@ -214,6 +212,31 @@ export class StudioRouter extends CustomRouter {
         res.send(totalEnv)
       })
     )
+
+    this.router.get(
+      '/labeling.js',
```

Comment:
being picky right now I know but can you rename this for `branding` for consistency and clarity

Reply:
Yeah good idea!